src/server.rs

```diff
@@ -273,6 +310,7 @@ fn read_opt(
         "last": is_last,
         "index": msg_max,
         "status": 200,
+        "request_id": request_id,
```

Comment:
why does it need to be sent back in the response?

Reply:
Not sure if it does or not, but since it's a 'pass-thru' i figured it would be good to do the full circuit. This way the recipient system can either verify or use the value for it's own reasons.